hoomd/hpmc/pytest/test_pair_user.py

```diff
@@ -89,14 +89,14 @@ def test_attaching(device, simulation_factory, two_particle_snapshot_factory):
 
 @pytest.mark.validate
 @pytest.mark.skipif(llvm_disabled, reason='LLVM not enabled')
-def test_kernel_parameters(simulation_factory, two_particle_snapshot_factory):
-    patch = hoomd.hpmc.pair.user.CPPPotential(r_cut=3,
+def test_kernel_parameters(simulation_factory, lattice_snapshot_factory):
```

Comment:
Is there a reason for switching to use `lattice_snapshot_factory` in more places?

Reply:
With the two particle snapshot factory, one of the ranks had no particles and failed to run the kernel. Thus the autotuner on that rank didn't change and the assert checking for changed values failed.